test/client/fixtures/sandbox/cookie-test.js

```diff
@@ -89,6 +91,73 @@ test('remove real cookie after browser processing', function () {
     strictEqual(nativeMethods.documentCookieGetter.call(document).indexOf(uniqKey), -1);
 });
 
+module('unnecessary synchronization cookies');
+
+test('different path', function () {
```

Comment:
I run tests on local machine and it failed. Should we prepare `document.cookie` before each test (clear or something else)?